test/system/250-systemd.bats

```diff
@@ -82,32 +80,34 @@ function service_cleanup() {
 
     # All good. Stop service, clean up.
     # Also make sure the service is in the `inactive` state (see #11304).
-    service_cleanup inactive
```

Comment:
looks like this is not actually doing what the comment is saying, https://github.com/containers/podman/issues/11304

Reply:
Well that just removes the regression test for this issue then (I understand that it did not actually worked before)? It looks like we should check for the unit status.

Reply:
Oh all right. I've taken the time to understand the issue and figure out the magic systemd incantations. I was not, however, able to test using podman v3.3.0 because that version is way too old to run on my system (I finally gave up on a `containers.conf` incompatibility). New check is my best guess at addressing the original intention.